app/sidekiq/update_view.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class UpdateView
+  include Sidekiq::Job
+
+  def perform
+    unless Rank.last
+      NewRankBookrankService.new.perform
+      sleep(5)
```

Comment:
why use `sleep` here?

Reply:
It will get a big bug if ` UpdateViewService.new.perform` executed while no data in Rank and BookRank so 
` NewRankBookrankService.new.perform` will fix this case.

im just needed to make sure that ` NewRankBookrankService.new.perform` absolutely done by dalayed for 5 second.

Is it too redundant code for top down code running languages krub :) (Im really agreed lol)